talisker/gunicorn.py

```diff
@@ -58,16 +58,8 @@ def prometheus_multiprocess_worker_exit(server, worker):
         multiprocess.mark_process_dead(worker.pid)
 
 
-class GunicornLogger(gstatsd.Statsd):
-    """Custom gunicorn logger to use structured logging.
-
-    Based on the statsd gunicorn logger, and also increases timestamp
-    resolution to include msec in access and error logs.
-    """
-    def __init__(self, cfg):
-        super(GunicornLogger, self).__init__(cfg)
-        if self.sock is not None:
-            self.sock.close()
```

Comment:
Yeah, we didn't use it. The base class opened a socket, but we didn't use it, so I just closed it to make sure.

This change just drops the use of that base class altogether, as we have diverged/customised significantly